docs/Features/XMLReferencesFeatures.md

```diff
@@ -0,0 +1,122 @@
+# XML References Features
+
+## References
+
+Given this [docbook](https://docbook.org/) XML file sample:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN" "http://www.docbook.org/xml/4.4/docbookx.dtd">
```

Comment:
Add a note that "resolve external entities" must be enabled for this schema to work.

Reply:
Good catch, fixed.